client/app/ui-components/button/button.component.js

```diff
@@ -1,75 +1,43 @@
 angular.module("managerApp")
-    .component("cuiButton", {
-        template: `
-            <button class="oui-button oui-button_{{ $ctrl.type }}"
-                    type="submit"
-                    data-ng-bind="$ctrl.text"
-                    data-ng-click="$ctrl.onClick()"
-                    data-ng-disabled="$ctrl.disabled"></button>
-        `,
-        bindings: {
-            text: "<",
-            onClick: "&",
-            disabled: "<",
-            type: "@"
-        }
-    })
     .component("cuiSubmitButton", {
         template: `
-            <button class="oui-button oui-button_primary"
-                    type="submit"
-                    data-ng-bind="$ctrl.text"
-                    data-ng-click="$ctrl.onClick()"
-                data-ng-disabled="$ctrl.disabled"></button>
+            <oui-button variant="primary"
+                type="submit"
+                data-on-click="$ctrl.onClick()"
+                data-disabled="$ctrl.disabled"
+                data-text="{{::$ctrl.text}}"
+                data-aria-label="{{::$ctrl.ariaLabel}}"
+                data-id="{{::$ctrl.id}}"
+                data-name="{{::$ctrl.name}}">
+            </oui-button>
         `,
         bindings: {
             text: "<",
             onClick: "&",
-            disabled: "<"
+            disabled: "<?",
+            ariaLabel: "@?",
+            id: "@?",
+            name: "@?"
         }
     })
     .component("cuiCancelButton", {
         template: `
-            <button class="oui-button oui-button_secondary"
-                    type="button"
-                    data-ng-bind="$ctrl.text"
-                    data-ng-click="$ctrl.onClick()"
-                    data-ng-disabled="$ctrl.disabled"></button>
+            <oui-button variant="secondary"
```

Comment:
Sometimes you stick with `data-` as prefix and sometimes not. Could you please stay uniform? :)